ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetDifficultyCalculators.java

```diff
@@ -67,6 +67,17 @@ private MainnetDifficultyCalculators() {}
         return periodCount > 1 ? adjustForPeriod(periodCount, difficulty) : difficulty;
       };
 
+  public static DifficultyCalculator<Void> DIFFICULTY_BOMB_REMOVED =
```

Comment:
Hot take: could this instead become `NO_DIFFICULTY_BOMB` and have` HOMESTEAD` wrap it?  So this comes first then homestad with the bomb adjustment, making no difficulty the base case?